### ISSUE TITLE
UPDATE THE DOCUMENTATION

### ISSUE THREAD

**Issue by achristlex** [MicrosoftDocs/bing-docs#107]
Good god, I cannot put into words how frustrating it is trying to find ANYTHING out when it comes Visual Search. There are at least 3 different legacy links to various API versions and while one does redirect to the updated version, even that one contains an entire Client API library that no longer functions at all: https://learn.microsoft.com/en-us/bing/search-apis/bing-visual-search/quickstarts/sdk/visual-search-client-library-python, **_and it was updated less than a month ago!_** I just spent an hour talking to a confused (though helpful) support worker and he had no idea. I can only hope this is not indicative of the rest of Azure.

The state of the documentation is already confusing enough, compared to something like Google's offer on the same feature: https://cloud.google.com/vision/docs/detecting-web.

Please, PLEASE do better.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 06c9df5e-0cb2-ed9c-68a1-1838a487e143
* Version Independent ID: c3edf6bd-17d3-edc7-fe26-121a6064b5c0
* Content: [Bing Visual Search API quickstarts - Bing Search Services](https://learn.microsoft.com/en-us/bing/search-apis/bing-visual-search/quickstarts/quickstarts)
* Content Source: [bing-docs/bing-visual-search/quickstarts/quickstarts.md](https://github.com/MicrosoftDocs/bing-docs/blob/main/bing-docs/bing-visual-search/quickstarts/quickstarts.md)
* Service: **bing-search-services**
* Sub-service: **bing-visual-search**
* GitHub Login: @swhite-msft
* Microsoft Alias: **scottwhi**

**Comment by shriramBaNW** [comment]
I completely agree! It's outdated and i've been unable to receive any support whatsoever!

**Comment by yitongYao** [comment]
I agree with you! The process of applying for and using the Bing Search API has been extremely frustrating. I don't know what to do, and the documentation contains many errors. hope they can do better
